.swiftlint.yml

```diff
@@ -25,7 +25,6 @@ opt_in_rules:
   - missing_docs
   - modifier_order
   - multiline_arguments
-  - multiline_function_chains
```

Comment:
I removed this linter rule because it led to unnecessary (imo) line breaks when chaining or waiting on futures.

For example, the following would not be allowed:
```swift
coll.insertOne([:]).flatMap { _ in
     coll.insertOne([:])
}.flatMap { _ in
     coll.insertOne([:])    
}.flatMap { _ in
     coll.insertOne([:])
}

try coll.watch(
    pipeline,
    options: options
).wait()
```
They would have to be written as:
```swift
coll.insertOne([:]).flatMap { _ in
     coll.insertOne([:])
}
.flatMap { _ in
     coll.insertOne([:])    
}
.flatMap { _ in
     coll.insertOne([:])
}

try coll.watch(
    pipeline,
    options: options
)
.wait()
```
Which I think is too spaced out. What do you think?